styles/style.css

```diff
@@ -120,18 +157,23 @@ p {
 /* hero */
 
 .hero {
-  background-image: url("header-bike.jpg");
+  background-image: url("../images/header-bike.jpg");
+  background-size: cover;
   background-color: var(--grey-light);
+  padding: 6rem 3rem;
```

Comment:
Very nice, good that you used rem over px - this was a CYF best practice last I heard